src/artrace.py

```diff
@@ -1209,7 +1209,13 @@ def model_tilt(slf, det, msarc, censpec=None, maskval=-999999.9,
         elif slf._argflag['trace']['orders']['tilts'].lower() == "spca":
              xspl = np.linspace(0.0, 1.0, msarc.shape[1])
              #yspl = np.append(0.0, np.append(arcdet[np.where(aduse)]/(msarc.shape[0]-1.0), 1.0))
-             yspl = np.append(0.0, np.append(polytilts[arcdet[np.where(aduse)],msarc.shape[1]/2],1.0))
```

Comment:
I'm pretty sure that this bug is not present in the interp method, and doesn't need to be resolved for the spline method. You might want to check, but I think your yspl method is the same as the more condensed form:
```
ycen = np.diag(polytilts[arcdet[np.where(aduse)], ordcen[arcdet[np.where(aduse)]]])
yspl = np.append(0.0, np.append(ycen, 1.0))
```
but you might want to check... (see line 1149-1150).